snuba/cli/migrations.py

```diff
@@ -0,0 +1,36 @@
+import click
+
+from snuba.migrations.runner import Runner
+from snuba.migrations.status import Status
+
+
+@click.group()
+def migrations() -> None:
+    """
+    Currently only for development use
+    """
+    click.echo("Warning: The migrations tool is currently only for development use\n")
+
+
+@migrations.command()
+def list() -> None:
+    """
+    Lists migrations and their statuses
+    """
+    runner = Runner()
+    for group, group_migrations in runner.show_all():
+        click.echo(group.value)
+        for migration_id, status, blocking in group_migrations:
+            symbol = {
+                Status.COMPLETED: "x",
+                Status.NOT_STARTED: " ",
+                Status.IN_PROGRESS: "?",
```

Comment:
I'd just write the status "COMPLETE", "IN PROGRESS", etc.
Or use some more explicit unicode symbols if you prefer symbols, it does not seem very clear to me that "x" means complete and "?" is in progress.

Reply:
I'll update to a combination of both so there's no room for confusion

Reply:
Couldn't get unicode to look great, I update to roughly match the output of django migrations, and always print the text for in progress.